### DISCUSSION TITLE
Connecting sensitivity run with prior predictive draws

### DISCUSSION THREAD

**Discussion by hyunjimoon** [Data4DM/BayesSD#30] [people; relating]
Can import parameter values through files
<img width="857" alt="image" src="https://user-images.githubusercontent.com/30194633/193299278-1c674664-9e84-46bd-8593-ea1e51216b5a.png">

**Comment by tomfid** [comment]
This option works if the distribution needed is available in Vensim. If you need some kind of customized distribution, you could write equations for that, and then use the sensitivity setup only to vary the NOISE SEED as a VECTOR input. Another option might be to create a payoff containing only the prior terms (no data), and run MCMC to generate a sample from the prior distribution.

**Comment by tomfid** [comment]
![image](https://user-images.githubusercontent.com/2151849/193881167-c76cca2c-4620-48c5-8c94-1587dc64c33e.png)
